configcat/manifest.json

```
@@ -0,0 +1,28 @@
+{
+  "integration_id": "configcat",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "categories": [
+    "notification"
+  ],
+  "creates_events": true,
+  "display_name": "ConfigCat",
+  "guid": "5e98d95a-519d-460c-945a-5e3a4e1f8d72",
+  "is_public": true,
+  "maintainer": "developer@configcat.com",
+  "manifest_version": "1.0.0",
+  "name": "configcat",
+  "public_title": "Datadog-ConfigCat integration",
```

```suggestion
  "public_title": "Datadog-ConfigCat Integration",
```

CI validation fails otherwise.

```
configcat/manifest.json... FAILED
  invalid `public_title`: Datadog-ConfigCat integration
```

https://dev.azure.com/datadoghq/integrations-extras/_build/results?buildId=11445&view=logs&j=910f82a4-622d-5ace-f877-8b75c045486c&t=57642e18-890a-5eef-b695-2b3c448c1261